exotica_python/src/Exotica.cpp

```diff
@@ -596,7 +596,7 @@ PYBIND11_MODULE(_pyexotica, module)
     kdlFrame.def("getTranslationAndAngleAxis", [](KDL::Frame* me) { return getFrameAsVector(*me, RotationType::ANGLE_AXIS); });
     kdlFrame.def("getTranslationAndQuaternion", [](KDL::Frame* me) { return getFrameAsVector(*me, RotationType::QUATERNION); });
     kdlFrame.def("getFrame", [](KDL::Frame* me) { return getFrame(*me); });
-    kdlFrame.def("inverse", (KDL::Frame(KDL::Frame::*)() const) & KDL::Frame::Inverse);
+    kdlFrame.def("inverse", (KDL::Frame (KDL::Frame::*)() const) & KDL::Frame::Inverse);
```

Comment:
We have now upgraded our formatting requirement to `clang-format-3.9`. These changes go away/need to be reverted when using the correct version.

Reply:
I ran 
```
find -name '*.cpp' -name '*.h' -o -name '*.hpp' | xargs clang-format-3.9 -style=file -i
```
in exotica root. 
Changes were not made to `Exotica.cpp`.  So I guess previously this file had not been formatted with `clang-format-3.9`.

Reply:
I rebased and the issue went away. Your new changes to address comments did not get formatting applied so are failing the build server check.